src/xcode/ENA/ENA/Source/Scenes/DynamicTableViewController/Views/DynamicTableViewBulletPointCell.swift

```diff
@@ -61,13 +62,15 @@ final class DynamicTableViewBulletPointCell: UITableViewCell {
 		layoutIfNeeded()
 	}
 
+	/// This configure will automatically set overwriteAttributes to true. So If you have already attributes in the attributedStrings you want to keep, you can use this configuration call. This will ensure, that your attributes are not overwritten internally
```

Comment:
"This will ensure, that your attributes are not overwritten internally"

Is this true, if `overwriteAttributes` is set to `true`?

Reply:
I refactored the naming for better understanding. Hope is clearer now.